apps/simple_history.py

```diff
@@ -0,0 +1,91 @@
+"""
+Copyright 2019 ShipChain, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+
+from django.core.serializers import serialize
+from django.db import models
+
+from simple_history.models import HistoricalRecords, ModelChange, ModelDelta
+
+
+RELATED_TO_LOCATION = ('shipment_to', 'shipment_from', 'shipment_dest', 'shipment_bill', )
+
+
+def get_user(request=None, instance=None):
+
+    if request:
+        return request.user.id
+
+    if instance:
+        if hasattr(instance, 'updated_by'):
+            return instance.updated_by
+        for related in RELATED_TO_LOCATION:
+            if hasattr(instance, related):
+                return getattr(instance, related).updated_by
+
+    return None
+
+
+def _model_to_dict(model):
+    return json.loads(serialize("json", [model]))[0]["fields"]
```

Comment:
I see that you copied this code from django-simple-history - if there are no changes, please just import the version from their library